cypress/support/step_definitions/dictionary/addConceptInDictionaryPage/addconceptInDictioanryPage.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import {
+  Given,
+  Then,
+  When
+} from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from "../../../utils";
+
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+});
+
+When('the user clicks the "Add concepts" button', () => {
+  cy.findByTitle("Add concepts").click();
+});
+
+Then('the user should be on the "Import existing concept" page', () => {
+  cy.url().should("contain", "/orgs/CIEL/sources/CIEL/concepts/");
+});
+
+Given('the user is on the "Import existing concept" page', () => {
+  cy.url().should("contain", "/orgs/CIEL/sources/CIEL/concepts/");
+});
+
+When('the user clicks on the row for "Serum"', () => {
+  cy.findByText("Serum").click();
+});
+
+When('the user clicks the "Add selected to dictionary" button', () => {
+  cy.findByTitle("Add selected to dictionary").click();
+});
+
+Then('the "Serum" concept should be added to the dictionary', () => {
+  cy.visit(`/orgs/CIEL/sources/CIEL/concepts/?addToDictionary=/users/${getUser()}/collections/${getDictionaryId()}/Serum/`)
```

Comment:
Isn't it should be something like this?

```suggestion
  cy.url().should('include', `/orgs/CIEL/sources/CIEL/concepts/?addToDictionary=/users/${getUser()}/collections/${getDictionaryId()}/Serum/`)
```

Reply:
Check the following:
https://docs.cypress.io/api/commands/url#Assert-the-URL-is-http-localhost-8000-users-1-edit

Reply:
Is there any other way to determine whether the concept has been added? instead of checking the URL? A user-specific way.

ex: How a user knows when it has been created? Is it displays a success message? or it displays the created concept?

Reply:
Since for this case am importing the concept from the already existing concepts, it just displays the imported concept in the dictionary.